public/plugins/ckeditor/skins/bootstrapck/scss/components/_toolbar.scss

```diff
@@ -0,0 +1,317 @@
+   be pasted. */
+.cke_button_disabled .cke_button_icon {
+    opacity: 0.3;
+}
+
+.cke_hc .cke_button_disabled {
+    opacity: 0.5;
+}
+
+a.cke_button_on {
+    &:hover, &:focus, &:active {
+    }
+}
+
+a.cke_button_off, a.cke_button_disabled {
+    &:hover, &:focus, &:active {
+        background: $primary-lighter;
+    }
+}
+
+/* The icon which is a visual representation of the button. */
+.cke_button_icon {
+    cursor: inherit;
+    background-repeat: no-repeat;
+    margin-top: 1px;
+    width: 16px;
+    height: 16px;
+    float: left;
+    display: inline-block;
+}
+
+.cke_rtl .cke_button_icon {
+    float: right;
+}
+
+.cke_hc .cke_button_icon {
+    display: none;
+}
+
+/* The label of the button that stores the name of the feature. By default,
+   labels are invisible. They can be revealed on demand though. */
+.cke_button_label {
+    display: none;
+    padding-left: 3px;
+    margin-top: 1px;
+    line-height: 18px;
+    vertical-align: middle;
+    float: left;
+    cursor: default;
+    color: $gray-dark;
+}
+
+.cke_rtl .cke_button_label {
+    padding-right: 3px;
+    padding-left: 0;
+    float: right;
+}
+
+.cke_hc .cke_button_label {
+    padding: 0;
+    display: inline-block;
+    font-size: 12px;
+}
+
+/* The small arrow available on buttons that can be expanded
+   (e.g. the color buttons). */
+.cke_button_arrow {
+    /* Arrow in CSS */
+    display: inline-block;
+    margin: 8px 0 0 1px;
+    width: 0;
+    height: 0;
+    cursor: default;
+    vertical-align: top;
+    border-left: 3px solid transparent;
+    border-right: 3px solid transparent;
+    border-top: 3px solid #474747;
+}
+
+.cke_rtl .cke_button_arrow {
+    margin-right: 5px;
+    margin-left: 0;
+}
+
+.cke_hc .cke_button_arrow {
+    font-size: 10px;
+    margin: 3px -2px 0 3px;
+    width: auto;
+    border: 0;
+}
+
+/* The vertical separator which is used within a single toolbar to split
+   buttons into sub-groups. */
+.cke_toolbar_separator {
+    float: left;
+    background-color: $hr-border;
+    margin: 4px 2px 0;
+    height: 16px;
+    width: 1px;
+}
+
+.cke_rtl .cke_toolbar_separator {
+    float: right;
+}
+
+.cke_hc .cke_toolbar_separator {
+    width: 0;
+    border-left: 1px solid;
+    margin: 1px 5px 0 0px;
+}
+
+/* The dummy element that breaks toolbars.
+   Once it is placed, the very next toolbar is moved to the new row. */
+.cke_toolbar_break {
+    display: block;
+    clear: left;
+}
+
+.cke_rtl .cke_toolbar_break {
+    clear: right;
+}
+
+/* The button, which when clicked hides (collapses) all the toolbars. */
+.cke_toolbox_collapser {
+    width: 12px;
+    height: 11px;
+    float: right;
+    margin: 11px 0 0;
+    font-size: 0;
+    cursor: default;
+    text-align: center;
+
+    border: 1px solid #a6a6a6;
+    border-bottom-color: #979797;
+
+    border-radius: $border-radius;
+
+    background: #e4e4e4;
+    &:hover {
+        background: #ccc;
+    }
+    &.cke_toolbox_collapser_min {
```

Comment:
Selector `cke_toolbox_collapser_min` should be written in hyphenated BEM (Block Element Modifier) format